include/rsl/algorithm.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <algorithm>
+
+namespace rsl {
+
+/** @file */
+
+/**
+ * @brief Determine if a collection contains a value
+ *
+ * @param collection Input collection
+ * @param value      Value to test for in the collection
+ *
+ * @return True if the value is found in the collection
+ */
```

Comment:
For the sake of consistency we probably need to document the template parameter

Reply:
:shrug: I think it should be deduced... I'm a fan of less noise in the docs if a user is unlikely to specify it.